docs/blog/2025-03-19-using-angular-with-rspack.md

```diff
@@ -0,0 +1,166 @@
+
+We believe that a large portion of Angular applications still using Webpack will be able to migrate to Angular Rspack without issue.
+
+There are some limitations and missing features that are currently being worked on and on the roadmap to support. They have been listed at the bottom of this article.
+
+## Migrating from Angular Webpack to Angular Rspack
+
+We have worked hard to make the migration process as smooth as possible.
+A new generator has been added to the `@nx/angular` package called `convert-to-rspack` which will help you migrate your Angular applications from Webpack to Rspack.
+
+The steps are very simple:
+
+1. Run `nx migrate latest` to update your workspace to the latest version of Nx.
+2. Run `nx g @nx/angular:convert-to-rspack` to migrate your Angular application to Rspack.
+
+We have also created a [guide](/recipes/angular/rspack/migrate-from-webpack) that walks you through the process step-by-step.
+Even if you're currently using the Angular CLI, it's as simple as first running `npx nx init` in your workspace and then running `npx nx convert-to-rspack`.
+
+## Using Angular Rspack
+
+You'll notice a that your `build` and `serve` targets have been removed from your project, and are instead inferred by the `@nx/rspack/plugin`.
+You should also notice a new `rspack.config.ts` file in your project which looks something like this:
+
+```ts
+import { createConfig } from '@nx/angular-rspack';
+export default createConfig(
+  {
+    options: {
+      root: __dirname,
+      outputPath: {
+        base: '../../dist/apps/app',
+      },
+      index: './src/index.html',
+      browser: './src/main.ts',
+      polyfills: ['zone.js'],
+      tsConfig: './tsconfig.app.json',
+      assets: [
+        './src/favicon.ico',
+        './src/assets',
+        {
+          input: './public',
+          glob: '**/*',
+        },
+      ],
+      styles: ['./src/styles.scss'],
+      scripts: [],
+    },
+  },
+  {
+    production: {
+      options: {
+        outputPath: {
+          base: '../../dist/apps/app-prod',
+        },
+        index: './src/index.prod.html',
+        browser: './src/main.prod.ts',
+        tsConfig: './tsconfig.prod.json',
+      },
+    },
+  }
+);
+```
+
+{% callout type="info" title="createConfig Information" %}
+The `createConfig` function is used to create an Rspack configuration object setup for Angular applications.
+You can read more about it [here](/nx-api/angular-rspack/documents/create-config).
+{% /callout %}
+
+### Build and Serving your Application
+
+You can now run `nx build app` and `nx serve app` to build and serve your application via Rspack.
+
+### Running a configuration
+
+To run the build with the `production` configuration:
+
+```bash
+NGRS_CONFIG=production nx build app
+```
+
+{% callout type="info" title="NGRS_CONFIG" %}
```

Comment:
I would have this as part of the normal flow without a callout necessarily.

Reply:
Ok, no problem!